spec/examples/record/udt_spec.rb

```diff
@@ -0,0 +1,70 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path('../spec_helper', __FILE__)
+
+describe Cequel::Record do
```

Comment:
I wasn't too sure what to put here since I didn't really define a model to represent .a UDT since `Cassandra::UDT` handles it pretty well